### PR TITLE
CIV-8918 Claim Submission - Assign Claim

### PR DESCRIPTION
**Pull request by kenneth-hmcts**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-8918

### Change description ###
Lip case assignment

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseEvent/systemUpdateCUI-nonprod.json [hunk 1]
```diff
@@ -4,5 +4,11 @@
     "CaseEventID": "RESET_PIN",
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "CRU"
```
Comment (TatsyMaslova) [inline ccd-definition/AuthorisationCaseEvent/systemUpdateCUI-nonprod.json:6]: this should be for release 2 so should be in a new file - systemUpdate-R2-CUI.json

File: ccd-definition/CaseEvent/Camunda/BusinessProcessEvents-CUI-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
```
Comment (TatsyMaslova) [inline ccd-definition/CaseEvent/Camunda/BusinessProcessEvents-CUI-nonprod.json:3]: For release2 we agreed to have R2 post fix so would be BusinessProcessEvents-R2-CUI-nonprod.json. so it does not interfere with release1 which is still in progress

File: ccd-definition/AuthorisationCaseEvent/systemUpdateCUI-nonprod.json (hunk 1 above)
Comment (kenneth-hmcts) [reply ccd-definition/AuthorisationCaseEvent/systemUpdateCUI-nonprod.json]: Updated as suggested

File: ccd-definition/CaseEvent/Camunda/BusinessProcessEvents-CUI-nonprod.json (hunk 2 above)
Comment (kenneth-hmcts) [reply ccd-definition/CaseEvent/Camunda/BusinessProcessEvents-CUI-nonprod.json]: Updated as suggested

**Review by neeta-hmcts — APPROVED** [review]

**Review by TatsyMaslova — APPROVED** [review]

**Review by rajakm — APPROVED** [review]

**Pull request merged**